.github/workflows/conformance.yml

```diff
@@ -0,0 +1,16 @@
+on:
+  # manual dispatch only while the conformance test suite is under rapid development
+  workflow_dispatch:
+
+name: CI
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the client wrapper
```

Comment:
nit: We don't call it wrapper elsewhere here. IMO calling it conformance client would be clearer.